src/model/trust.page.model.ts

```diff
@@ -40,13 +40,13 @@ interface TrustHistoricalBeneficialOwnerFormCommon {
 }
 
 type IndividualTrusteesFormCommon = {
-  trustee_id?: string,
+  trusteeId?: string,
   type: RoleWithinTrustType,
   forename: string,
   surname: string,
-  date_of_birth_day: string,
-  date_of_birth_month: string,
-  date_of_birth_year: string,
+  dateOfBirthDay: string,
+  dateOfBirthMonth: string,
+  dateOfBirthYear: string,
   nationality: string,
```

Comment:
Why are we renaming these variables? 
If they do need renaming for whatever reason then the other vars with multiple words should follow the same standard. Here there is a miss of camelCase and snake_case

Reply:
Steve mentioned the same issue. It is as a result of how the UI makes a post request with a mixture of both, as I reuse 2 modules that do both. This is a candidate for refactoring but will make the PR larger than needs to be and I will have to fix lots of models. Hence, used it the way it is. Many models have done same but that will require changing in a separate ticket.